src/Features/CSharp/Portable/Completion/CompletionProviders/CrefCompletionProvider.cs

```diff
@@ -319,6 +319,7 @@ private static TextSpan GetCompletionItemSpan(SourceText text, int position)
                                 RefKind.Ref => "ref ",
                                 RefKind.Out => "out ",
                                 RefKind.In => "in ",
+                                RefKind.RefReadOnlyParameter => "ref readonly ",
```

Comment:
Not related to this PR: should `scoped` be here too?

Reply:
I think probably not, `scoped` is not important in `cref`s similarly to parameter names. In fact, it's a parsing error to put `scoped` there.